packages/angular/src/app/pages/common-profile/common-profile.component.html

```diff
@@ -0,0 +1,109 @@
+<dx-scroll-view class="view-wrapper-scroll">
+  <div class="view-wrapper">
+    <dx-toolbar>
+      <dxi-item location="before">
+        <div class="header-text">User Profile</div>
+      </dxi-item>
+      <dxi-item location="after" locateInMenu="never">
+        <dx-button
+          text="Cancel"
+          stylingMode="contained"
+        ></dx-button>
+      </dxi-item>
+      <dxi-item location="after" locateInMenu="never">
+        <dx-button
+          [disabled]="!isDataChanged"
+          text="Save"
+          icon="save"
+          type="default"
+          stylingMode="contained"
+          (click)="save()"
+        ></dx-button>
+      </dxi-item>
+    </dx-toolbar>
+
+    <dx-load-panel
+      [visible]="isLoading"
+      [showPane]="false"
+      container=".view-wrapper-scroll"
+      [position]="{ of: 'common-profile' }"
+    ></dx-load-panel>
+    <div *ngIf="!isLoading" class="cards-container">
+      <profile-card class="basic-info-card"
+                    title="Basic Info"
+                    [colCount]="4"
+                    [cardData]="profileData"
+                    [items]="basicInfoItems"
+                    (dataChanged)="dataChanged()"
+      >
+        <div class="basic-info-top-item" top-item>
+          <form-photo
+            [link]="profileData?.image"
+            [editable]="true"
+            [size]="80"
+          ></form-photo>
+          <div>
+            <div class="title-text">{{profileData.name}}</div>
+            <div class="subtitle-text with-clipboard-copy">ID: {{profileData.id}}
+              <dx-button icon="copy"
+                         class="copy-clipboard-button"
+                         (click)="copyToClipboard(profileData.id)"
+                         [activeStateEnabled]="false"
+                         [focusStateEnabled]="false"
+                         [hoverStateEnabled]="false"
+              ></dx-button>
```

Comment:
Let's wrap inner text in something here:
```suggestion
            <div class="subtitle-text with-clipboard-copy">
              <span>ID: {{profileData.id}}</span>
              <dx-button icon="copy"
                         class="copy-clipboard-button"
                         (click)="copyToClipboard(profileData.id)"
                         [activeStateEnabled]="false"
                         [focusStateEnabled]="false"
                         [hoverStateEnabled]="false"
              ></dx-button>
```